main.go

```diff
@@ -299,6 +314,30 @@ func setupIndexes(ctx context.Context, mgr ctrl.Manager) {
 	}
 }
 
+func setupWebhookTLSConfigs(mgr ctrl.Manager, tlsMinVersion, tlsCipherSuites string) {
```

Comment:
WDYT about renaming the func to `calculateTLSOpts(....) ([]func(*tls.Config), error)

This way we can use the result easily to set it on manager.Options once we bump to CR v0.13.1 (which will hopefully happen soon & before merge of this PR - I'm on it)

Reply:
CAPI main now uses v0.13.1